pymongoexplain/__main__.py

```diff
@@ -48,6 +49,14 @@ def new_func(self: Collection, *args, **kwargs):
     setattr(Collection, old_func_name, make_func(old_func, old_func_name))
 
 if __name__ == '__main__':
-    for file in sys.argv[1:]:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("inputs", action="store", nargs=1)
+    parser.add_argument("--arg", "--arg", help="add arguments to explained "
+                                               "script",
+                        action="store", nargs="+")
```

Comment:
As Prashant mentioned here: https://github.com/mongodb-labs/pymongoexplain/pull/14#pullrequestreview-452664078

We don't need `--args`.

Reply:
I removed --args, so now it currently follows the pattern ```python -m pymongoexplain <script path> <script arg1> <script arg2>...<script arg n>```